.captain/config.yaml

```diff
@@ -7,3 +7,11 @@ test-suites:
         github-step-summary:
     results:
       path: report.xml
+  backwards-compatibility:
```

Comment:
Might want to call this `captain-cli-backwards-compatibility` in case we want backwards compatibility in any other project